### ISSUE TITLE
"pointer-events" property is not considered valid

### ISSUE THREAD

**Issue by uglow** [rossPatton/stylint#52]
I get a warning when using the `pointer-events` property in CSS class:

```css
.my-class
    &:after
      pointer-events: none
```

Config: 
```json
{
  "alphabetical": true,
  "borderNone": true,
  "brackets": false,
  "colons": false,
  "colors": false,
  "commaSpace": true,
  "commentSpace": true,
  "cssLiteral": false,
  "depthLimit": 3,
  "duplicates": true,
  "efficient": true,
  "enforceBlockStyle": false,
  "enforceVarStyle": true,
  "extendPref": "@extend",
  "globalDupe": true,
  "indentSpaces": 2,
  "leadingZero": true,
  "maxWarnings": 10,
  "maxWarningsKill": false,
  "mixed": true,
  "namingConvention": "lowercase-dash",
  "parenSpace": false,
  "placeholders": true,
  "semicolons": false,
  "universal": true,
  "valid": true,
  "whitespace": true,
  "zeroUnits": true,
  "zIndexDuplicates": false,
  "zIndexNormalize": false
}
```

Output:
```console
Warning:  property is not valid
File: path/to/file.styl
Line: 163: pointer-events: none; 
```

**Comment by rossPatton** [comment]
thanks! this is an easy fix, i'll update this thread when it's out

**Comment by rossPatton** [comment]
should be in 0.9.3
